src/main/java/com/cloudbees/jenkins/Cleaner.java

```diff
@@ -66,7 +73,7 @@ protected void doRun() throws Exception {
             for (GHRepository repo : r.resolve()) {
                 try {
                     removeHook(repo, Trigger.all().get(DescriptorImpl.class).getHookUrl());
-                    LOGGER.fine("Removed a hook from "+r+"");
+                    LOGGER.info("Removed a hook from "+r+"");
```

Comment:
regardless is is better form to use parameterized logging - and the concatenation of the final empty string looks bogus!